### ISSUE TITLE
Remove import statements in CSS

### ISSUE THREAD

**Issue by astronomersiva** [pythonindia/inpycon2019#42]

**Comment by Rohithgilla12** [comment]
Hey, do you mean this import statement
`@import url('https://fonts.googleapis.com/css?family=Montserrat:400,500,600,700|Muli:400,600,700');`
Or are there anymore?
Thanks

**Comment by astronomersiva** [comment]
@Rohithgilla12, yes. We should change that to make use of link tags and use font related CSS properties.

**Comment by Rohithgilla12** [comment]
Any updates on the change

**Comment by astronomersiva** [comment]
Left a comment on it last week. Please check.

**Comment by Rohithgilla12** [comment]
Yeah i modified it

**Comment by astronomersiva** [comment]
@Rohithgilla12 the PR still has the _site directory committed to it. Can you please check again and ensure that you have pushed the right changes?
